app/src/main/java/com/example/randomuserapplication/UserDetailsFragment.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.randomuserapplication
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
```

Comment:
this is not needed

Reply:
I removed it o. Lemme show you a screenshot

Reply:
![screen2a](https://github.com/Obehi234/Random-User-Generator/assets/48982222/bc08342f-f03f-4145-b81f-49bdcc1d90ee)

Reply:
my working tree is up to date as well